checks/linux.py

```diff
@@ -0,0 +1,172 @@
+from .vars import *
+from .utils.utils import *
+
+
+def getSessionTypeLine(lines):
+    sessionType = search('Session Type:', lines)
+    if len(sessionType) > 0:
+        return sessionType[0]
+
+
+def getWindowSystemLine(lines):
+    windowSystem = search('Window System:', lines)
+    if len(windowSystem) > 0:
+        return windowSystem[0]
+
+
+def checkDistro(lines):
+    isDistroNix = search('Distribution:', lines)
+
+    if len(isDistroNix) <= 0:
+        return
+
+    distro = isDistroNix[0].split()
+    distro = distro[2:]
+    distro = ' '.join(distro)
+
+    return [LEVEL_INFO, distro, ""]
+
+
+def checkFlatpak(lines):
+    isFlatpak = search('Flatpak Runtime:', lines)
+
+    if len(isFlatpak) > 0:
+        return [LEVEL_INFO, "Flatpak",
+                "You are using the Flatpak. Plugins are available as Flatpak extensions, which you can find in your Distribution's Software Center or via <code>flatpak search com.obsproject.Studio</code>. Installation of external plugins is not supported."]
+
+
+def checkSnapPackage(lines):
+    isDistroNix = search('Distribution:', lines)
+
+    if len(isDistroNix) <= 0:
+        return
+
+    if '"Ubuntu Core"' in isDistroNix[0]:
+        return [LEVEL_WARNING, "Snap Package",
+                "You are using the Snap Package. This is a community-supported modified build of OBS Studio; please file issues on the <a href=\"https://github.com/snapcrafters/obs-studio/issues\">Snapcrafters GitHub</a>.<br><br>OBS may be unable to assist with issues arising out of the usage of this package and therefore recommends following our <a href=\"https://obsproject.com/download#linux\">Install Instructions</a>."]
+
+
+def checkWayland(lines):
+    isDistroNix = search('Distribution:', lines)
+    isFlatpak = search('Flatpak Runtime:', lines)
+
+    if (len(isDistroNix) <= 0) and (len(isFlatpak) <= 0):
+        return
+
+    sessionTypeLine = getSessionTypeLine(lines)
+    if not sessionTypeLine:
+        return
+
+    if 'wayland' not in sessionTypeLine:
+        return
+
+    if len(isDistroNix) > 0:
+        if '"Ubuntu" "20.04"' in isDistroNix[0]:
+            return [LEVEL_CRITICAL, "Ubuntu 20.04 under Wayland",
+                    "Ubuntu 20.04 does not provide the needed dependencies for OBS to capture under Wayland.<br> So OBS is able to capture only under X11/Xorg."]
```

Comment:
```suggestion
                    "Ubuntu 20.04 does not provide the needed dependencies for OBS to capture under Wayland.<br> Capture  will only function under X11/Xorg."]
```

Reply:
Done.
God bless 20.04 is going EOL in 4 months :pray: